stationd.py

```diff
@@ -0,0 +1,554 @@
+        gpiopin.set_direction(gpio.OUT)
+    return gpiopin
+
+
+def success_response(command_obj):
+    command = command_obj.command
+    message = f'SUCCESS: {command[0]} {command[1]} {command[2]}\n'
+    command_obj.sock.sendto(message.encode('utf-8'), command_obj.addr)
+    log(command_obj, message)
+
+
+def status_response(command_obj, status):
+    message = status
+    command_obj.sock.sendto(message.encode('utf-8'), command_obj.addr)
+    message = message.replace('\n', ', ')
+    log(command_obj, message)
+
+
+def temp_response(command_obj, temp):
+    message = f'temp: {str(temp)}\n'
+    command_obj.sock.sendto(message.encode('utf-8'), command_obj.addr)
+    log(command_obj, message)
+
+
+# Exceptions
+class No_Change(Exception):
+    def __init__(self, command_obj):
+        self.command_obj = command_obj
+
+    def send_response(self):
+        command = self.command_obj.command
+        message = f'WARNING: {command[0]} {command[1]} {command[2]} No Change\n'
+        self.command_obj.sock.sendto(message.encode('utf-8'), self.command_obj.addr)
+        log(self.command_obj, message)
+
+
+class Molly_Guard(Exception):
+    def __init__(self, command_obj):
+        self.command_obj = command_obj
+
+    def send_response(self):
+        message = 'Re-enter the command within the next 20 seconds if you would like to proceed\n'
+        self.command_obj.sock.sendto(message.encode('utf-8'), self.command_obj.addr)
+        log(self.command_obj, message)
+
+
+class PTT_Conflict(Exception):
+    def __init__(self, command_obj):
+        self.command_obj = command_obj
+
+    def send_response(self):
+        command = self.command_obj.command
+        message = f'FAIL: {command[0]} {command[1]} {command[2]} PTT Conflict\n'
+        self.command_obj.sock.sendto(message.encode('utf-8'), self.command_obj.addr)
+        log(self.command_obj, message)
+
+
+class Max_PTT(Exception):
+    def __init__(self, command_obj):
+        self.command_obj = command_obj
+
+    def send_response(self):
+        command = self.command_obj.command
+        message = f'Fail: {command[0]} {command[1]} {command[2]} Max PTT\n'
+        self.command_obj.sock.sendto(message.encode('utf-8'), self.command_obj.addr)
+        log(self.command_obj, message)
+
+
+class PTT_Cooldown(Exception):
+    def __init__(self, command_obj, seconds):
+        self.command_obj = command_obj
+        self.seconds = seconds
+
+    def send_response(self):
+        message = f'WARNING: Please wait {self.seconds} seconds and try again\n'
+        self.command_obj.sock.sendto(message.encode('utf-8'), self.command_obj.addr)
+        log(self.command_obj, message)
+
+
+class Invalid_Command(Exception):
+    def __init__(self, command_obj):
+        self.command_obj = command_obj
+
+    def send_response(self):
+        message = f'FAIL: Invalid Command\n'
+        self.command_obj.sock.sendto(message.encode('utf-8'), self.command_obj.addr)
+        log(self.command_obj, message)
```

Comment:
Make a base exception for this project (something like `StationdError`) and derived all other custom exception from that one.